Makefile

```diff
@@ -91,6 +91,7 @@ docker-mirror-read:
 undeploy-fybrik:
 	$(TOOLBIN)/helm uninstall fybrik-crd --namespace $(KUBE_NAMESPACE)
 	$(TOOLBIN)/helm uninstall fybrik --namespace $(KUBE_NAMESPACE)
+	$(TOOLBIN)/kubectl delete namespace $(KUBE_NAMESPACE) || true
```

Comment:
Do not delete fybrik-system. This namespace is not controlled by fybrik/fybrik-crd releases. For example, vault is deployed in fybrik-system and should not be re-deployed each time.